### ISSUE TITLE
Add support for producing x86 and x64 assemblies if input assembly is AnyCpu

### ISSUE THREAD

**Issue by batzen** [MeikTranel/NXPorts#9]
For snoop i don't want separate projects for x86 and x64.
The exporter i wrote for snoop creates both from an input assembly that is AnyCpu and i would like to contribute that code here.

I guess we'd need some msbuild properties to express this.
Any suggestions for that?

**Comment by MeikTranel** [comment]
I designed this with the idea that platform stuff is entirely controlled by msbuild. 
If we were to build runtime specific dlls from AnyCPU we would also have to deal with ARM32 and ARM64/Pref32 and all that. I'd prefer to stay in the system in place.

With MSBuild.Sdk.Extras you can run builds in multi-RID mode using the InnerLoop/OuterLoop system already in place for the *TargetFrameworks* property. The following project file would build x64 and x86 binaries on dotnet build / msbuild / visual studio build.

```xml
<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
  <PropertyGroup>
    <TargetFrameworks>net452</TargetFrameworks>
    <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
    <ExtrasBuildEachRuntimeIdentifier>true</ExtrasBuildEachRuntimeIdentifier>
  </PropertyGroup>
</Project>
```

More info here: https://github.com/onovotny/MSBuildSdkExtras#creating-per-runtimeidentifier-packages

**Comment by MeikTranel** [comment]
ahh... i see... Snoop uses the old csproj format?
Would you like help with moving the sdk-based project files or have you already tried and discovered an issue holding you back?

**Comment by batzen** [comment]
You have to look at the develop branch of snoop. It's already converted to the sdk format. ;-)

If you don't want to produce multiple dlls with nxports, nxports should force a specific rid to be present instead of doing it's work based on 32bit prefered flag, shouldn't it?

**Comment by MeikTranel** [comment]
Yes that would definitely be something we should do. I'm thinking about an error on build whenever either runtimeidentifier nor platformtarget is set. I just discovered some more multi-RID specific issues.
So this is something i can look at as well.

**Comment by MeikTranel** [comment]
Closed in favor of #12
